version.txt

```diff
@@ -1 +1 @@
-0.0.1-beta-006
+0.0.1-beta-007
```

Comment:
```suggestion
0.0.1-beta-006
```

Can we revert this for now? I'd like to increment the version / nuget package after I fix up a few warnings that this PR introduces so for now the changes will be unpublished on nuget itself

Reply:
Actually nvm we can accept it as is since we're in beta still